schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id integer NOT NULL,
```

Comment:
- [ ] [OPTIONAL] Kindly mark the `id` column as auto increment so you don't have to insert the value of that column manually.

```SQL
id int NOT NULL IDENTITY(1,1)
```